controller/controller_server/messages.py

```diff
@@ -3,8 +3,12 @@
 ObjectIdError_message = "Wrong {0} id format : {1}"
 
 # validation error messages
-invalid_secret_message = "invalid secret was passed"
-secret_missing_message = 'secret is missing'
+invalid_secrets_message = "invalid secrets was passed"
+invalid_topology_key_message = "invalid topology key was passed: {}"
+invalid_topology_value_message = "invalid topology value was passed: {}"
+invalid_system_id_value_message = "invalid system id value was passed: {}"
+invalid_non_json_parameter_message = "invalid non json parameter was passed: {}."
+secrets_missing_message = 'secrets is missing'
```

Comment:
from the user perspective (and grammatically), this should be singular

Reply:
done